### PR TITLE
Twis44

### PR DESCRIPTION
**Pull request by larsbergstrom**
r? @jdm @Manishearth @nox 

Please add any other notable PRs (just push to the `twis44` branch!) and feel free to land. Thanks1

[Review on Reviewable](https://reviewable.io/reviews/servo/blog.servo.org/49)

### CODE REVIEW

File: _posts/2015-12-21-twis-44.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- [Beomjin Kim](https://github.com/vwvww)
+- [Florian Strübe](https://github.com/fstr)
+- [Jigesh Mehta](https://github.com/jitendra29)
+- [Jinwoo Ahn](https://github.com/jinwooahn)
+- [John DeSilva](https://github.com/Aesthetikx)
+- [Mathieu Agopian](https://github.com/magopian)
+- [Mike Atkins](https://github.com/apechimp)
+- [Olaf Buddenhagen](https://github.com/antrik)
+- [Ori Avtalion](https://github.com/salty-horse)
+- [ReadmeCritic](https://github.com/ReadmeCritic)
```
Comment (Manishearth) [inline _posts/2015-12-21-twis-44.md]: Not human.

**Comment by Manishearth** [comment]
So far lgtm, will look for more notable PRs later today.

**Comment by Ms2ger** [comment]
http://blogs.s-osg.org/community-driven-wayland-support-servo/
http://pcwalton.github.io/blog/2015/12/21/drawing-css-box-shadows-in-webrender/

File: _posts/2015-12-21-twis-44.md (hunk 1 above)
Comment (larsbergstrom) [inline _posts/2015-12-21-twis-44.md]: No love for the robots? :-)

**Pull request merged**